zebra/zebra_fpm.c

```diff
@@ -2029,11 +2031,24 @@ static int zfpm_init(struct thread_master *master)
 	return 0;
 }
 
+static int zfpm_fini(void)
+{
```

Comment:
Should we take care of the below two or these are taken care else where for closing the FPM connection ?

1) close the zfpm_g->sock  socket ? 
2) reset zfpm_g->ibuf and zfpm_g->obuf ?

Reply:
There are all sorts of memory leaks in the fpm code upon shutdown.  I made no attempt to actually resolve those as that I believe a crash is worse than a memory leak on shutdown.  imo someone needs to take ownership of the fpm code and fix this issue

Reply:
@donaldsharp I will investigate to find such leaks and will take care of fixing. Could you please list if you have any thing on top of you mind ?

I will go ahead and merge after CI.

Reply:
the socket is not closed, ibuf and obuf are not freed there is a mac hash table that is not freed.  That is just off the top of my head